src/index.html

```diff
@@ -11,7 +11,141 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+          method="post"
+          onsubmit="onSubmit()">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="surname">
+          Surname:
+          <input type="text"
+                 id="surname"
+                 name="surname"
+                 autocomplete="off"
+                 required>
```

Comment:
```suggestion
          <input
            type="text"
            id="surname"
            name="surname"
            autocomplete="off"
            required
          >
```
Fix formatting everywhere